src/components/Sidebar/Sidebar.vue

```diff
@@ -14,13 +14,28 @@
       </div>
 
       <div class="sb-sidebar__mobile-logo" @click="closeSidebar">
-        <img
-          v-if="logo"
-          class="sb-custom-logo"
-          :src="logo"
-          alt="Custom Sidebar Logo"
-        />
-        <SbSidebarLogo v-else variant="dark" />
+        <SbLink
+          v-if="logoDestinationUrl"
+          as="router-link"
+          :to="logoDestinationUrl"
+        >
+          <img
+            v-if="logo"
+            class="sb-custom-logo"
+            :src="logo"
+            alt="Custom Sidebar Logo"
+          />
+          <SbSidebarLogo v-else variant="dark" />
+        </SbLink>
+        <template v-else>
```

Comment:
I don't think it's necessary to wrap these elements inside a template tag if you use `v-if-else` for the image

Reply:
Oh I see, thanks for the explanation my friend 🚀